### PR TITLE
Fixing DOC #49 & #50

### PR DESCRIPTION
**Pull request by spirillen**
This should be fixing issue [#49](https://mypdns.org/spirillen/PyFunceble/-/issues/49) and #[50](https://mypdns.org/spirillen/PyFunceble/-/issues/50)

- Enhance the --local description
- Add note for log output in separated folder

Credit:
- ZeroDot1

Be careful for typos as it was a bit late while doing this

### CODE REVIEW

File: PyFunceble/cli/system/launcher.py [hunk 1]
```diff
@@ -676,6 +676,14 @@ def generate_percentage_file(parent_dirname: str) -> None:
             ):
                 break
 
+            generate_percentage_file(protocol["destination"])
```
Comment (funilrys) [inline PyFunceble/cli/system/launcher.py:679]: This one change shouldn't be there. What the hell is happening here ?

File: .gitlab/issue_templates/Documentation.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Description
```
Comment (funilrys) [inline .gitlab/issue_templates/Documentation.md:1]: I'm refusing to maintain this .gitlab directory. PyFunceble - until I change my mind - will only be maintained on GitHub. 

Right now GitLab is unusable for me. I have over 500+ \@\mentions across GitHub and GitLab. GitHub gives me inbox transparency. GitLab gives me headache. 

I'm ready to provide support for GitLab CI. But I'm not ready to accept this change here.

**Review by funilrys — CHANGES_REQUESTED** [review]

File: PyFunceble/cli/system/launcher.py (hunk 1 above)
Comment (spirillen) [reply PyFunceble/cli/system/launcher.py:679]: This is not the changes I made.... https://mypdns.org/spirillen/PyFunceble/-/commit/a2ff7278884be2c6cbda05cb3d753ef505611446

File: .gitlab/issue_templates/Documentation.md (hunk 2 above)
Comment (spirillen) [reply .gitlab/issue_templates/Documentation.md:1]: This is not the changes I made.... https://mypdns.org/spirillen/PyFunceble/-/commit/a2ff7278884be2c6cbda05cb3d753ef505611446

**Review by spirillen — COMMENTED** [review]
There should not have been any changes to 

- `PyFunceble/cli/system/launcher.py`
- `.gitlab/issue_templates/Documentation.md`

Only changes should be to `docs/usage/index.rst` and `docs/usage/terminal.rst`

**Comment by spirillen** [comment]
Found where the problem occurs

This MR is right: https://mypdns.org/spirillen/PyFunceble/-/merge_requests/47 and the one that should have made here. The problem then happens when I merge the `dev` branch  into the `4.x-doc branch` here https://mypdns.org/spirillen/PyFunceble/-/merge_requests/48

File: .gitlab/issue_templates/Documentation.md (hunk 2 above)
Comment (spirillen) [reply .gitlab/issue_templates/Documentation.md:1]: AAhhh now I can remember this one, This one is mostly intended for my use in my doc repo. not as such for the public

> Right now GitLab is unusable for me. I have over 500+ @\mentions across GitHub and GitLab. GitHub gives me inbox transparency. GitLab gives me headache.

I a knowledge this is that GH rules on this point as GL i more bound to mails than a internal "mailbox"

You can try and see my number
image

I'm sticking to the mailbox, to find new relevant issues.

**Comment by spirillen** [comment]
Replaced by https://github.com/funilrys/PyFunceble/pull/284

**Pull request closed without merging**